### DISCUSSION TITLE
PGE editor display scaling [Windows 11]

### DISCUSSION THREAD

**Discussion by CatoWongtc** [WohlSoft/PGE-Project#609] [Q&A] [answered]
I recently got a new ASUS PC with Windows 11 as default and reinstalled SMBX Beta 5. The PGE editor UI doesn't look right compared to the one on my previous Windows 10 PC. My current recommended system display resolution is 2560 x 1600 with a 150% recommended scale. Everything is too big (such as the UI icons, subwindows, and the main screen). This is with "Enable auto-scaling on high-DPI monitors" option checked in the Application Settings (see attached results). 
<img width="1280" alt="Screenshot 2024-04-10 184537" src="https://github.com/WohlSoft/Moondust-Project/assets/165170556/daeb3156-4095-402a-b921-e15529ba33ed">

When I unchecked the option, everything became too small (see attached results). 
<img width="1280" alt="Screenshot 2024-04-10 201948" src="https://github.com/WohlSoft/Moondust-Project/assets/165170556/752a1fb0-aeb5-4b1e-a63f-f6380da8b714">

I was looking for sizes somewhere in between. 
Additionally, the "Check for updates" option detected a new build, but when I went to the official site and downloaded it, the download got blocked by Google. When I tried using the cmd option in the PGE folder to update, it returned "Files not found" on step 3/4

**Comment by Wohlstand** [comment]
Hello! This is an old problem of Qt (the set of GUI libraries are used by Moondust Editor and components). I will need to try to produce builds with using of newer Qt versions, but:
- On X2 you won't get my updates at all. You will need to wait when `Emral` will backport these updates to the X2 Devkit (the fork of Moondust Devkit). Right now I don't have any contacts with him (Our business relationship just sucks).
- My (mainstream) version is not compatible to X2 Beta 5 (it's only can be used and updated for Beta 4 and older).

**Reply by CatoWongtc** [reply]
I don't think that's the problem. I did the same with B4 and the scaling issue still exists. My preferred GUI size could be achieved by setting my display resolution from 2560 x 1600 to 1920 x 1200. That would mean that the "Auto-scale on high DPI monitors" checkbox only provides one scaling, non-adjustable resolution for the editor, which is too big for me.

**Comment by CatoWongtc** [comment, marked as answer]
This seemed to work. A bit blurry but is in the right size that display every tab normally.
<img width="478" alt="Screenshot 2024-04-12 165247" src="https://github.com/WohlSoft/Moondust-Project/assets/165170556/a01d8793-0d25-4feb-a210-816981238239">
